web/app/css/tap.css

```diff
@@ -18,3 +18,20 @@
     color: rgba(52, 137, 206, 0.906);
   }
 }
+
+.tap-more-info {
+  padding-right: 50px;
+
+  & .tap-info-section {
+    padding: 24px 0;
```

Comment:
the expanded section feels a bit spaced out, what do you think about something like:
`padding: 2px 0;`

![screen shot 2018-09-13 at 4 35 18 pm](https://user-images.githubusercontent.com/236915/45521508-18ccd800-b773-11e8-9b3c-5117a781986d.png)

Reply:
it's true! I'll make it more compact. how about `4px 0` :P?